docs/SRS/SRS.tex

```diff
@@ -640,72 +640,72 @@ \subsection{Functional Requirements}
 \medskip
 \noindent \textbf{Requirement ID:} FR1\\
 \textbf{Use Case:} 1\\
-\textbf{Description:} The system must provide a user-friendly interface for users to enter their email and password when creating an account.\\
-\textbf{Rationale:} This requirement ensures that users can easily provide their registration information.\\
-\textbf{Fit Criterion:} Users should be able to successfully enter their email and password without encountering errors during the registration process.\\
+\textbf{Description:} The system must provide a clear and intuitive interface for account creation, where users can easily enter their email and password.\\
```

Comment:
some thing like "minimize user effort and reduce the likelihood of registration abandonment" might make a bit more unambiguous?

Reply:
resolved